src/idd/idd.chart.js

```diff
@@ -95,17 +91,18 @@
             });
         }
     }
-
-    this.onChildrenChanged = function (arg) {
-        if (arg.type == "add") {
+    this.fireChildrenChanged = function (propertyName) {
+        if (propertyName.type == "add") {
             if (that.xDataTransform) {
-                arg.plot.xDataTransform = that.xDataTransform;
+                propertyName.plot.xDataTransform = that.xDataTransform;
```

Comment:
I suggest to given different name to the parameter `propertyName`, probably `arg`, because it is not the name at all.